diagnostic.md

```diff
@@ -5,74 +5,85 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+to manage data (store created and updated data, send and receive data)
```

Comment:
Good, also it allows multiple users to interact from different devices while on the same app.